packages/core/src/actors/reducer.ts

```diff
@@ -9,22 +9,26 @@ import { isSCXMLEvent } from '../utils';
  * @returns An actor behavior
  */
 
-export function fromReducer<TState, TEvent extends EventObject>(
+export function fromReducer<TState extends {}, TEvent extends EventObject>(
```

Comment:
I don't think that we have to prescribe the constraint of `TState` here. The user should have complete freedom here
```suggestion
export function fromReducer<TState, TEvent extends EventObject>(
```